src/data/nat/fib.lean

```diff
@@ -96,4 +97,75 @@ begin
     mono }
 end
 
+/-- https://proofwiki.org/wiki/Consecutive_Fibonacci_Numbers_are_Coprime -/
```

Comment:
On these lines I'd add a few more words, like `See https://proofwiki.org/wiki/Consecutive_Fibonacci_Numbers_are_Coprime` or `Subsequent Fibonacci numbers are coprime, see https://proofwiki.org/wiki/Consecutive_Fibonacci_Numbers_are_Coprime` so it makes more sense when viewed in the doc.